.github/actions/upload_test_artifacts/action.yaml

```diff
@@ -1,16 +1,43 @@
 name: Test Artifact Upload
-description: Uploads test archives to GCS for on-device tests.
+description: Uploads test archives for on-device and on-host tests.
+inputs:
+  on_host:
+    description: "Indicates if on-host test artifacts should be uploaded."
+    required: true
+  on_device:
+    description: "Indicates if on-device test artifacts should be uploaded."
+    required: true
+  test_artifacts_key:
+    description: "Artifact key used to store on-host test artifacts."
+    required: true
 runs:
   using: "composite"
   steps:
     - name: Set up Cloud SDK
+      if: inputs.on_device == 'true'
       uses: isarkis/setup-gcloud@40dce7857b354839efac498d3632050f568090b6 # v1.1.1
-    - name: Upload Artifacts to GCS
+    - name: Upload On-Host Test Artifacts to GCS
```

Comment:
Should this name be "On-Device Test Artifacts" ?

Reply:
Good catch, yes. Let me fix.